### PR TITLE
OCLOMRS-159: A user should be able to Login with username and password

### PR DESCRIPTION
**Pull request by kmunene**
# JIRA TICKET NAME:
[A user should be able to Login with username and password](https://issues.openmrs.org/browse/OCLOMRS-159)

# Summary:
A user should be able to Login with username and password

### CODE REVIEW

File: src/redux/actions/auth/authActions.js [hunk 1]
```diff
@@ -3,13 +3,24 @@ import { login, loginFailed, loginStarted, logout } from './authActionCreators';
...
-const loginAction = ({ username, token }) => async (dispatch) => {
+const loginAction = ({ username, password }) => async (dispatch) => {
   try {
-    const url = `${BASE_URL}/users/${username}/`;
+    const data = {
+      username,
+      password,
+    };
+
+    // Get user's token
```
Comment (alexmochu) [inline src/redux/actions/auth/authActions.js]: An observation. Please be consistent with comments. But this good job.

Comment (dkayiwa) [reply src/redux/actions/auth/authActions.js]: In regards to the above comment, can you take a look at this? https://improvingsoftware.com/2011/06/27/5-best-practices-for-commenting-your-code/

Comment (kmunene) [reply src/redux/actions/auth/authActions.js]: @dkayiwa Thanks for the resource. I will work on it according.

Comment (kmunene) [reply src/redux/actions/auth/authActions.js]: @alexmochu Thank you for your observation. I will read on the best practices and conventions for writing comments.

**Pull request merged**